src/java/org/apache/cassandra/hints/HintsDispatchExecutor.java

```diff
@@ -205,21 +205,27 @@ private final class DispatchHintsTask implements Runnable
         private final UUID hostId;
         private final RateLimiter rateLimiter;
 
-        DispatchHintsTask(HintsStore store, UUID hostId)
+        DispatchHintsTask(HintsStore store, UUID hostId, boolean isTransfer)
```

Comment:
I don't see where this is used.

Reply:
Good catch. It's now wired up in `TransferHintsTask#transfer()`.